### DISCUSSION TITLE
[Bug] run open-webui in docker and get an error about port already in use

### DISCUSSION THREAD

**Discussion by zydmtaichi** [open-webui/open-webui#5136] [General]
I run the docker cmd below to map the ollama service port to inner docker env on ubuntu.
`docker run -d -p 3000:8080 -p 127.0.0.1:11434:11434 -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always registry.cn-hangzhou.aliyuncs.com/kin-images/open-webui:git-18ae586`

but when the ollama service on, the docker can't be started successfully. Why the open webui can't shart the 11434 port with ollama?

**Comment by zydmtaichi** [comment]
if I run ollama first, then i got an error for open-webui as below.
```
Error response from daemon: driver failed programming external connectivity on endpoint open-webui (ad070638625527538d4deb5222315c50a2472b2a954b8623c7f5e2b697ecff98): failed to bind port 127.0.0.1:11434/tcp: Error starting userland proxy: listen tcp4 127.0.0.1:11434: bind: address already in use
Error: failed to start containers: a8b891a3a425
```

**Comment by zydmtaichi** [comment]
@justinh-rahb hi, this issue is related to disintegration between ollama and open-webui, pls don't close on bug issue

**Reply by silentoplayz** [reply]
It's not a bug until proven otherwise. This looks like user error to me. Port 11434 is already in use.
